pandas/formats/format.py

```diff
@@ -26,12 +26,14 @@
 from pandas.core.base import PandasObject
 from pandas.core.index import Index, MultiIndex, _ensure_index
 from pandas import compat
-from pandas.compat import (StringIO, lzip, range, map, zip, reduce, u,
+from pandas.compat import (StringIO, lzip, range, map, zip, u,
                            OrderedDict, unichr)
 from pandas.util.terminal import get_terminal_size
 from pandas.core.config import get_option, set_option
 from pandas.io.common import _get_handle, UnicodeWriter, _expand_user
 from pandas.formats.printing import adjoin, justify, pprint_thing
+from pandas.formats.excel import ExcelFormatter  # for downstream # NOQA
```

Comment:
do we actually *need* to import this? I would rather just change the references to import directly ``ExcelFormatter` (from ``pandas.formats.excel``) wherever they are.